defaults/severities.yaml

```diff
@@ -99,9 +99,9 @@ CassandraStressEvent: CRITICAL
 GeminiStressEvent: CRITICAL
 ScyllaServerStatusEvent: NORMAL
 BootstrapEvent: NORMAL
-RepairEvent: NORMAL
+RepairEvent: DEBUG
 JMXServiceEvent: NORMAL
 InstanceStatusEvent.STARTUP: WARNING
 InstanceStatusEvent.REBOOT: WARNING
 InstanceStatusEvent.POWER_OFF: WARNING
-CompactionEvent: NORMAL
+CompactionEvent: DEBUG
```

Comment:
why are these being set as DEBUG, and not like either NORMAL?

Reply:
idea to do not throw them into the sct.log

Reply:
AFAICT @roydahan wanted to get them removed completely, not hidden from the eye (and not where it doesn't belong to)